windows/s2e.sln

```diff
@@ -14,10 +14,6 @@ EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "pdbparser", "pdbparser\pdbparser.vcxproj", "{3A1CDCF6-4B82-41C0-83AF-55E1D915AB3E}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{205799D9-3F67-4C14-B269-BAF6D63F09A5}"
-	ProjectSection(SolutionItems) = preProject
```

Comment:
Does this mean that the instructions in https://github.com/S2E/guest-tools/tree/master/windows regarding code style are no longer applicable? If so, the readme should be updated and kept in sync with these changes

Reply:
They are still applicable. I realized resharper is not free, so left the old settings.

Reply:
It's fine if the file you mention is not in the project. You have to import it manually.

Reply:
Okie-dokie